verification/curator-service/ui/cypress/integration/components/BulkCaseForm.spec.ts

```diff
@@ -55,6 +55,10 @@ describe('Bulk upload form', function () {
         cy.contains('www.bulksource.com');
         cy.contains('sourceEntryId');
         cy.contains('superuser@test.com');
+        cy.get(
+            //Upload ID
+            '[name="case-data"] > .makeStyles-grid-75 > :nth-child(8)',
```

Comment:
this seems kinda brittle, any way we can tag this with data-testid or something more stable?

Reply:
Can do -- I'll tree up from `Data upload ID` (since `data-testid` doesn't seem to work out of the box with the element we're using in ViewCase).